src/mytardis_client/endpoints.py

```diff
@@ -1,60 +1,194 @@
 """Information about MyTardis endpoints"""
 
+from typing import Literal, Optional
+
 from pydantic import BaseModel
 
 from src.mytardis_client.objects import MyTardisObject
 
+MyTardisEndpoint = Literal[
```

Comment:
Should this be only valid endpoints or only endpoints we want the ingestion scripts to be using?
if it's the former /group and /schema are both valid endpoints (and I might end up using /schema to validate RO-Crate metadata before ingestion)

Reply:
Good point, it wasn't totally clear in my head. I think I assumed there were some I didn't know about, and would add as they came up, so I've added those, thanks